api/v1beta1/foundationdbcluster_types.go

```diff
@@ -1808,10 +1817,13 @@ type DataCenter struct {
 // ContainerOverrides provides options for customizing a container created by
 // the operator.
 type ContainerOverrides struct {
-	// EnableLivenessProbe defines if the sidecar should have a livenessProbe in addition
-	// to the readinessProbe. This setting will be enabled per default in the 1.0.0 release.
+	// EnableLivenessProbe defines if the sidecar should have a livenessProbe.
+	// This setting will be ignored on the main container.
```

Comment:
You removed the `This setting will be enabled per default in the 1.0.0 release.` was that intended?

Reply:
We don't generally document future defaults, and I think it would be better to document them in a systematic way.